src/fmu/sumo/utilities/sim2sumo.py

```diff
@@ -0,0 +1,423 @@
+"""Export with metadata"""
+import re
+from pathlib import Path
+import logging
+import importlib
+import argparse
+from inspect import signature
+import pandas as pd
+import ecl2df as sim2df
+import ecl2df
+from pyarrow import Table
+from fmu.config.utilities import yaml_load
+from fmu.dataio import ExportData
+from fmu.sumo.uploader.scripts.sumo_upload import sumo_upload_main
+
+
+def _define_submodules():  #  -> Tuple(tuple, dict):
+    """Fetch all submodules
+
```

Comment:
What is the intention of this? E.g. why do we look for submodules?

Reply:
We need it for easy access to all datatypes, in stead of hard coding it in, it is detected with this function